### PR TITLE
[WIP] Add label for inputs, accessibility review

### PR DESCRIPTION
**Pull request by benjaminapetersen**
- Adds label for the key:
```html
<label for="key-{{$index}}" class="sr-only">{{keyPlaceholder}}</label>
```
and label for the value:
```html
<label for="value-{{$index}}" class="sr-only">{{valuePlaceholder}}</label>
```
- Added `tabindex="0"` to move row, delete row and add row, **however** am still testing to see if i can trigger these actions with the keyboard.  Looking into default accessibility of `ng-click` [here](https://docs.angularjs.org/guide/accessibility), as well as ngAria.

Couple additional things to note, these items are already included (@rhamilto creds): 

- `role="button"` is on links
- There is already `aria-hidden="true"  aria-grabbed="false"` on the icons when present
- There is `aria-label="Move row"` on the drag icon
- There is `aria-label="Delete row"` on the delete icon
- There is `aria-label="Add row"` on the add icon

Things outstanding on accessibility:

- We are not using `aria-dropeffect`, briefly reading up on `aria-grabbed` and `aria-dropeffect` [here](https://www.sitepoint.com/accessible-drag-drop/).

@jwforres @spadgett

### CODE REVIEW

File: src/views/key-value-editor.html [hunk 1]
```diff
@@ -12,6 +12,8 @@
         class="form-group key-value-editor-input"
         ng-class="{ 'has-error' :  (forms.keyValueEditor['key-' + $index].$invalid) }">
 
+        <label for="key-{{$index}}" class="sr-only">{{keyPlaceholder}}</label>
```
Comment (spadgett) [inline src/views/key-value-editor.html]: We're going to have ID conflicts if more than one editor is on the same page (e.g. one for env vars and one for labels).

Comment (spadgett) [inline src/views/key-value-editor.html]: I'd also give it a more specific prefix. `key-1` and `value-1` are so general they might collide with other unrelated stuff on the page.

Comment (benjaminapetersen) [inline src/views/key-value-editor.html]: Agree, I'm already using a unique id for the set focus, which will support multiple editors on the page, so I can update these to use the same Id (but a better prefix).

**Comment by benjaminapetersen** [comment]
Update: got keyboard events working for `delete` as well.  Sort is not supported at this point, but looking into it.

**Comment by benjaminapetersen** [comment]
@spadgett updated the unique ID generation.
@jwforres I think this is def a good step in the right direction.

File: src/views/key-value-editor.html [hunk 2]
```diff
@@ -196,6 +202,9 @@
       ng-if="(!cannotAdd) && addRowLink">
       <a
         href=""
+        role="button"
+        tabindex="0"
```
Comment (jwforres) [inline src/views/key-value-editor.html]: remove
